composer.json

```diff
@@ -3,15 +3,35 @@
     "description": "Symfony bundle for information about your Messenger queues and transports",
     "type": "symfony-bundle",
     "require": {
+        "php": "^7.1.3",
         "symfony/framework-bundle": "^4.4|^5.0",
-        "symfony/messenger": "^4.4|^5.0"
+        "symfony/messenger": "^4.4|^5.0",
+        "symfony/twig-bundle": "^4.4|^5.0"
+    },
+    "require-dev": {
+        "roave/security-advisories": "dev-master",
+        "symfony/phpunit-bridge": "^5.0",
+        "friendsofphp/php-cs-fixer": "^2.16"
     },
     "license": "MIT",
     "autoload": {
         "psr-4": {
-            "KaroIO\\MessengerMonitorBundle\\": ""
+            "KaroIO\\MessengerMonitorBundle\\": "src/"
+        },
+        "exclude-from-classmap": [
+            "src/Test/"
+        ]
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "KaroIO\\MessengerMonitorBundle\\": "tests/",
+            "KaroIO\\MessengerMonitorBundle\\Test\\": "src/Test"
```

Comment:
I would move the one class in this directory somewhere into `tests/` - or even just embed that class at the bottom of any tests that need it. Then we can remove all of this and the `exclude-from-classmap`